docs/releases.rst

```diff
@@ -5,6 +5,14 @@
 ======================
 
 
+tmt-1.52.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Files submitted during a test via
+:ref:`tmt-file-submit</stories/features/report-log>` or beakerlib's
+rlFileSubmit are now a part of main test result as logs.
```

Comment:
```suggestion
``rlFileSubmit`` are now a part of main test result as logs.
```

Reply:
@bajertom this seems to be a clear fix, just resolve the comments maybe if no more discussion needed next time?